core/vm/eips.go

```diff
@@ -32,6 +32,8 @@ func EnableEIP(eipNum int, jt *JumpTable) error {
 		enable1884(jt)
 	case 1344:
 		enable1344(jt)
+	case 2315:
```

Comment:
Should we have this as an option? Does this work for us?

Reply:
Do you know of anyone who con led some light into this?

Reply:
@mcortesi

Reply:
missing the context here?
So there's an EIP 2315, but we don't have an cli option to enable it? Or is the question wether we should ahve the code for 2315?

To me, we should add all code from EIPs since probably we will have them enabled on a future HF. And they should be disable for now, which probalby is already true since they are enabled using a chainConfig flag.

Reply:
I agree, this is just the code, I believe it wasn't activated yet